source/FlareComponent.js

```diff
@@ -1,372 +1,411 @@
-import React from "react";
-import {ActorLoader, Graphics} from "flare";
-import {vec2, mat2d} from "gl-matrix";
-import Logo from "./flare_logo.svg";
-
-export class FlareComponent extends React.Component 
-{
-    constructor(props)
-    {
-        super(props);
-		this.state = {hasError:false, isLoading:false, hasActor:false};
-
-		this._RuntimeGraphics = null;
-		this._ActorInstance = null;
-		this._RuntimeActor = null;
-		this._IsRendering = false;
-		this._LastAdvanceTime = Date.now();
-		this._RuntimeAnimationIndex = -1;
-		this._AnimationTime = 0;
-		this._OverrideAnimationPosition = null;
-		this._LastReportedAnimationPosition = -1;
-		this._AdvanceSpeed = 0.0;
-		this._TargetAdvanceSpeed = 0.0;
-        this._ViewTransform = mat2d.create();
-        
-        this.setCanvasRef = (ref) =>
-        {
-            this._Canvas = ref;
-        };
+import React, { createRef } from "react";
+import { ActorLoader, Graphics } from "flarejs/build/Flare.min.js";
```

Comment:
Was the submodule not resolving properly for you? I prefer to use the submodule here as it keeps all the relevant code in one tidy repo. Check .babelrc for how it resolves.

Reply:
it's been a while, so I'm not sure, honestly. might have just been my editor complaining about the import?